guides/accessibility/page-template-considerations.md

```diff
@@ -55,27 +53,26 @@ module('Acceptance | posts', function(hooks) {
 
   test('visiting /posts', async function(assert) {
     await visit('/posts');
-    assert.equal(document.title, 'Posts - Site Title');
+    assert.equal(document.title, 'Messages - Titre du site');
   });
 });
 ```
 
-## Skip Navigation Links
-
-A skip navigation link, or skip link, is a useful feature for users who want to bypass content that is repeated on multiple pages (i.e., a site header). This can especially helpful to users with assistive technology, who have to browse website content in a more linear fashion, but it can also be useful for power users who prefer to navigate websites only using a keyboard.
+## Liens pour sauter la navigation
 
-<img width="675px" title="Skip Main Content Example" alt="A visual representation of how the skip link works in the browser" src="/images/accessibility/page-template-considerations/skip-main-content.png"/>
+Un lien pour sauter la navigation (_skip navigation link_ ou _skip link_) est une fonctionnalité utile aux utilisateurs qui souhaitent passer le contenu répétés sur plusieurs pages (par exemple, l'entête du site). C'est particulièrement bienvenu pour les utilisateurs qui s'aident de technologies d'assistance, qui naviguent dans le contenu du site de manière plus linéaire, mais c'est aussi utile pour les utilisateurs avancés qui préfèrent naviguer sur le site uniquement avec le clavier.
 
-To implement a skip link in an application, add an anchor element as close as possible after the opening `<body>` element, and have it link to the beginning of the page's main content area.
+<img width="675px" title="Exemple de saut au contenu principal" alt="Une représentation visuelle de la manière dont le lien pour sauter la navigation fonctionne dans le navigateur" src="/images/accessibility/page-template-considerations/skip-main-content.png"/>
```

Comment:
In this case, it would rather be:
```suggestion
<img width="675px" title="Exemple où la navigation est sautée" alt="Une représentation visuelle de la manière dont le lien pour sauter la navigation fonctionne dans le navigateur" src="/images/accessibility/page-template-considerations/skip-main-content.png"/>
```
because we skip the navigation / skip to the content.